Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -139,13 +139,8 @@ WarpXParticleContainer::AddNParticles (int /*lev*/,
                                        const ParticleReal* vx, const ParticleReal* vy, const ParticleReal* vz,
                                        int nattr, const ParticleReal* attr, int uniqueparticles, amrex::Long id)
 {
-    // nattr is unused below but needed in the BL_ASSERT
-    amrex::ignore_unused(nattr);
-
     BL_ASSERT(nattr == 1);
```

Comment:
What is the purpose of this assert? It doesn't cause a runtime error if `nattr != 1` so I don't understand it's purpose. Seeing as this PR specifically allows `nattr > 1` is it fine to remove the assert?

Reply:
This assert can be deleted.